src/Domain/UseCase/CreateValetToken/CreateValetTokenResponse.ts

```diff
@@ -0,0 +1,15 @@
+import { Operation } from '../../Operation/Operation'
+import { ValetToken } from '../../ValetToken/ValetToken'
+
+export type ErrorResponse = {
```

Comment:
one class/type/interface per file

Reply:
Perhaps a mistake here was putting `ErrorResponse` before `CreateValetKeyResponse` -- I'll rearrange. "The most important thing at the top" is a rule I sometimes forget to ensure I'm following.

Reasons to not inline the `ErrorResponse`:
* keeps (inferred or not) type annotations shorter
* more descriptive
* it's clear that the success and error response types are symmetrical and determined by the value of the `success` field
* it's free